Environmental/DRAXIS/Python/cork_env_waterquality_yearly/elastic.py

```diff
@@ -0,0 +1,56 @@
+from elasticsearch import Elasticsearch
+
+
+class ElasticSearchClient:
+    def __init__(self, host: str, port: str):
+        self.es = Elasticsearch([{'host': host, 'port': port}])
+        self.index = None
+
+    def insert_doc(self, doc_: dict, id_: str = None) -> str:
+        res = self.es.index(index=self.index, body=doc_, id=id_)
+        return res['result']
+
+    @staticmethod
+    def define_geomapping():
```

Comment:
i would suggest we keep the same structure for all mappings... so maybe "define"_<es_type_definition>_"mapping".

so `define_geo_point_mapping`, `define_date_mapping` etc